docs/about-mono/releases/4.6.0.md

```diff
@@ -59,6 +59,44 @@ For custom implementations, the type should subclass the
 `Mono.Security.Interface.MonoTlsProvider` interface from the
 `Mono.Security` assembly and have a public parameterless constructor.
 
+### Mono symbolicate
+
+The previous version of mono-symbolicate was not created with the multiple symbols versions in mind.
+
+There were issues while symbolicating stacktraces with:
+ - IL offsets, as there were no way of knowing to which version of the managed assembly the IL refers to.
+ - Native offsets, as multiple .msym are generated and there were no way of directly linking the stack trace to those symbols.
+
+The problem of mapping a stack trace IL offsets to the correct managed assembly version was done by changing a stackframe such as:
+```
+  at StackTraceDumper.Main () [0x00000] in <filename unknown>:0
+```
+into one as:
+```
+  at StackTraceDumper.Main () [0x00000] in <cwa3ec873a838c42c29f4eba7cab7d1dfb>:0
+```
+Where the hash looking part in between `<>` is the module version ID (MVID) of the assembly module the method belongs to.
+
+The problem of mapping native offsets to the corrects .msym version was dome by replacing a stack frame such as:
+```
+  at StackTraceDumper.<Main>m__0 () <0x100e15d50 + 0x00029> in <filename unknown>:0
+```
+by on one as:
+```
+  at StackTraceDumper.<Main>m__0 () <0x100e15d50 + 0x00029> in <a3ec873a838c42c29f4eba7cab7d1dfb#b37a9d11247b811ffffdaff8090d8a59>:0
+```
+The new stack frame also displays the MVID but after the `#` it also displays what we named as AOTID, an unique identifier embedded into the AOT compiled assembly.
+
+With the changes above it is possible to know from the stack frames only which symbols versions are to be used in order to symbolicate it.
+
+A symbol directory was also introduced, its purpose is to store managed assemblies into subfolders called as MVID and native assemblies into subfolders called AOTID.
+
+`mono-symbolicate` is now able to retrieve the correct symbols versions from the stack trace, and to symbolicate it using the symbols in the symbol directory.
+
+The changes greatly improved `mono-symbolicate` usability and eliminated the versioning issues it had before.
+
+`mono-symbolicate` can be found [here](https://github.com/mono/mono/tree/master/mcs/tools/mono-symbolicate)
```

Comment:
please link to the 4.6.0 branch here so this link doesn't get stale if we e.g. move the tool in the future